src/task2/validators/user.ts

```diff
@@ -0,0 +1,39 @@
+import * as Joi from '@hapi/joi';
+import {
+  createValidator,
+  ValidatedRequestSchema,
+  ContainerTypes,
+} from 'express-joi-validation';
+
+const validator = createValidator({
+  passError: true,
+});
+
+const loginValidatator = Joi.string().trim().pattern(/^\S*$/);
```

Comment:
this regex won't allow users with a space in their username

Reply:
yep, that was an idea. But maybe it could be removed in that implementation